addons/stock_landed_costs/models/stock_landed_cost.py

```diff
@@ -132,7 +130,7 @@ def get_valuation_lines(self):
 
         for move in self.mapped('picking_ids').mapped('move_lines'):
             # it doesn't make sense to make a landed cost for a product that isn't set as being valuated in real time at real cost
-            if move.product_id.valuation != 'real_time' or move.product_id.cost_method not in ('average', 'fifo'):
+            if move.product_id.cost_method not in ('average', 'fifo'):
```

Comment:
When the valuation is the last record, assign line.price_unit less the accumulated value in value_split

https://github.com/Vauxoo/addons-vauxoo/blob/8.0/stock_landed_costs_average/model/stock_landed_costs.py#L592-L602